interface.js

```
@@ -0,0 +1,142 @@
+};
+
+function startBowling() {
+  hideIntro();
+  makeTable(bowling.players);
+  revealBoard();
+  updateCurrentPlayer();
+};
+
+function hideIntro() {
+  var x = document.getElementsByClassName("intro");
+  var i;
+  for (i= 0; i < x.length; i++)
+  {
+    x[i].style.display = "none"
+  };
+};
+function revealBoard() {
+  var x = document.getElementsByClassName("game-started");
+  for (let index = 0; index < x.length; index++) {
+    x[index].style.display = "block"
+  };
+};
+function makeTable(players) {
+  var perrow = 1, // 1 cells per row
+      count = 0, // Flag for current cell
+      table = document.createElement("table"),
+      row = table.insertRow(),
+      cell = row.insertCell();
+      cell.innerHTML = 'Names\\Frames';
+      for (let index = 1; index < 12; index++) {
+        cell =row.insertCell();
+        if(index < 11){
+          cell.innerHTML = `  ${index}  `
+        } else {
+          cell.innerHTML = 'Final Score'
+        }
+      };
+      row = table.insertRow();
+      for (var i of players) {
+        var cell = row.insertCell();
+        cell.innerHTML = i.name;
+        for (let index = 1; index < 12; index+= 1) {
+          cell = row.insertCell();
+          if(index < 11) {
+            cell.id = `${i.name}${index}`
+          } else {
+            cell.id = `${i.name}score`
+          }
+        };
+       count++;
+       if (count%perrow==0) {
+         row = table.insertRow();
+        }
+      };
+      
+      table.id = "score-board"
+      $("#game-board").append(table);
+};
+function addBallsToTable(ballOne, ballTwo) {
+  var cell = document.getElementById(`${bowling.getCurrentPlayer().name}${bowling.currentFrame}`)
+  if(ballOne + ballTwo < 10) {
+    cell.innerHTML= `${ballOne},${ballTwo}`
+  } else if(ballOne < 10 && ballOne + ballTwo === 10) {
+    cell.innerHTML= '/'
+  } else if(ballOne === 10) {
+    cell.innerHTML='X'
+  } else {
+    cell.innerHTML= 'Foul'
+  }
+;}
+function inputChecker(number) {
```

For things like this that should not happen, you could throw an error instead.